.github/pull_request_template.md

```diff
@@ -0,0 +1,7 @@
+Fixes #
+
```

Comment:
There should be a `Short Description:` section to highlight what the main goal is of the PR, beyond what's in the title